src/sentry/mediators/sentry_apps/updater.py

```diff
@@ -7,8 +7,9 @@
 
 from sentry.constants import SentryAppStatus
 from sentry.mediators import Mediator, Param
+from sentry.mediators.service_hooks import Updater as ServiceHookUpdater
```

Comment:
I like the pattern we've been doing elsewhere, where we import the module and call the class through that. eg.

```python
from sentry.mediators import service_hooks
service_hooks.Updater.run()
```

Seems explicit and easy to follow.